ui/src/app/components/summary/summary.component.scss

```diff
@@ -153,3 +153,7 @@ h3 {
     }
   }
 }
+.spanner-tab-link-summary{
```

Comment:
move this to styles.css and underline the text

Reply:
Updated